pkg/node/platform_description.go

```diff
@@ -0,0 +1,48 @@
+package node
+
+import (
+	"strconv"
+)
+
+type PlatformDescription struct {
+	imageName     string
+	daemonsetName string
+	labeltag      string
+	checkVersion
+}
+
+type checkVersion func(version string) bool
+
+var (
+	platforms = map[string]*PlatformDescription{
+		"default": {
+			imageName:     "csi-baremetal-node",
```

Comment:
Can `imageName` and `daemonsetName` be autogenerated?

Reply:
Remove these fields from datastruct. Now names are chosen with functions